index.html

```diff
@@ -760,11 +759,16 @@
 \begin{equation}
 \mathbf{F}_{82}(\mu) = \mathbf{F}_{\mathrm{Schlick}}(\mu) - \frac{\mu (1 - \mu)^6}{\bar{\mu}(1 - \bar{\mu})^6} \Bigl(\mathbf{F}_{\mathrm{Schlick}}(\bar{\mu}) - \mathbf{F}(\bar{\mu})\Bigr)
 \end{equation}
-where $\bar{\mu} = 1/7$, and $\mathbf{F}(\bar{\mu})$ is the desired metallic reflectivity at that "grazing edge" angle cosine corresponding roughly to $82^\circ$ (i.e. around silhouettes), ensuring $\mathbf{F}_{82}(\bar{\mu}) = \mathbf{F}(\bar{\mu})$. This desired edge reflectivity is user-specified as a fractional tint of the Schlick curve, i.e.
+where $\bar{\mu} = 1/7$, and $\mathbf{F}(\bar{\mu})$ is the desired metallic reflectivity at that "grazing edge" angle cosine corresponding roughly to $82^\circ$ (i.e. around silhouettes), ensuring $\mathbf{F}_{82}(\bar{\mu}) = \mathbf{F}(\bar{\mu})$. This desired edge reflectivity is user-specified as a fractional tint of the Schlick curve control via **`specular_color`**, i.e.
 \begin{equation}
-\mathbf{F}(\bar{\mu}) = \mathtt{specular\_weight} * \mathtt{specular\_color} * \mathbf{F}_\mathrm{Schlick}(\bar{\mu}) \ .
+\mathbf{F}(\bar{\mu}) = \mathtt{specular\_color} * \mathbf{F}_\mathrm{Schlick}(\bar{\mu}) \ .
 \end{equation}
-This formulation has the useful property that it reduces to the regular Schlick reflectivity at the default values of $\mathtt{specular\_weight}$ and $\mathtt{specular\_color}$. Note that the edge cannot be brighter than the standard Schlick term, but this is generally true in real metals. We consider this a benefit of this parametrization, as it makes it impossible to produce physically implausible metals with excessively bright edges.
+The final metallic Fresnel term we employ is then given by an overall multiplication by **`specular_weight`**, ensuring that entire metallic lobe is suppressed as the weight goes to zero:
```

Comment:
>Suppose an artist makes a material with a combination of dielectric and metal. Then after the fact, they way to tune the specular reflection with a uniform specular_weight. They are not going to observe the same behaviour on the dielectric and metal parts, which I think could be questionable.

I think actually it's more sensible now than it was before, since now the `specular_weight` multiplies the normal-incidence reflectance (i.e. F0) of both the dielectric and the metal.

Before, we had it only affecting the F82-tint color of the metal, but not the F0. I found that more inconsistent, since the F0 is the dominant effect. (Also, it didn't allow the metal lobe to be zeroed, as you noticed, so we have to have the weight multiply the entire lobe instead, which is more consistent with the dielectric F0 anyway).

The `specular_weight` multiplies the metallic lobe though, rather than operating via the IOR, so the effect is of course slightly different (suppression at grazing angles, rather than "tightening of the white highlight"). But we don't have an IOR control for the metal, so it's not possible to make them match to that extent.

Reply:
>I should also note, given the recent discussions on the metal-rough model, that there is a bit of inconsistency in how the metal layer is parametrised: other layers have a weight that acts as a coverage parameter. One could argue metalness is conceptually close to that role.

No, the metal is a component of the base, mixed with the "non-metallic base", according to the `base_metalness`. So `base_metalness` operates as a mix weight, not a coverage weight. Similarly for `transmission_weight` and `subsurface_weight`. 

A coverage weight only makes sense for a layer which can optionally "cover" another, i.e. the coat and fuzz layer.

Note that we also decided against trying to interpret `specular_weight` as the coverage weight of the diffuse base with dielectric (since the dielectric is not really functioning as a layer, it is really the base substrate in which the scattering media of the transmission/subsurface/glossy-diffuse slabs are embedded). Instead, `specular_weight` operates by modulating the IOR of the unified base dielectric.

Reply:
>Then after the fact, they way to tune the specular reflection with a uniform specular_weight. They are not going to observe the same behaviour on the dielectric and metal parts, which I think could be questionable.

I see what you mean though, if they have a textured metalness, then the single `specular_weight` affects both the metal and dielectric parts, which maybe they don't want. They'd have to modulate the weight by the metalness map. 

This goes back to our discussions about having a separate optional parametrization for metal. If you really want this use case of e.g.  metal and non-metal bits on the same object, you have to deal with blending of various maps to get that look, and then get issues in the transition regions because you are trying to represent a 50/50 mix of metal and dielectric with blended maps driving both the metal and dielectric BRDFs, instead of independent maps for the two BRDFs.

If that's a problem, the solution is something like the proposed additional metal params. But people don't like that because it adds complexity. My compromise approach is to accept that these issues are real, but treat this as a limitation of our model that we traded off for ease of use and simplicity. For better quality results, people can always do more elaborate things like use MaterialX to add layers of dielectric stuff on top of OpenPBR metal, say.

Reply:
Yes, I agree this is an improvement over the previous proposition, and I don't see a better alternative.
My concern is whether this could become a subject of complain, and if it does, can we still make a better proposal in a future without being locked by this decision.

That being said, if in the future we decide to go toward separate metal parameters, then this one parameter would likely be overridden for metal anyway, so I guess we're safe.